internal/core/Cargo.toml

```diff
@@ -23,14 +23,16 @@ libm = ["num-traits/libm", "euclid/libm"]
 # Allow the viewer to query at runtime information about item types
 rtti = []
 # Use the standard library
-std = ["euclid/std", "once_cell/std", "scoped-tls-hkt", "lyon_path", "lyon_algorithms", "lyon_geom", "lyon_svg", "instant", "image-decoders", "svg"]
+std = ["euclid/std", "once_cell/std", "scoped-tls-hkt", "lyon_path", "lyon_algorithms", "lyon_geom", "lyon_svg", "instant", "image-decoders", "svg", "clipboard"]
 # Unsafe feature meaning that there is only one core running and all thread_local are static.
 # You can only enable this feature if you are sure that any API of this crate is only called
 # from a single core, and not in a interrupt or signal handler.
 unsafe_single_core = []
 
 text_layout = []
 
+clipboard = ["copypasta"]
```

Comment:
Do we really need a feature for that. Can we not simply use "copypasta" as the feature.